### ISSUE TITLE
Prediction accuracy for example.py

### ISSUE THREAD

**Issue by zwang4** [mohaseeb/shaplets-python#4]
Hello,

I have ran example.py multiple times (200 epoch for training) for the GUN_point data set. I standardized the training/test data, and changed the code make sure it outputs a prediction label of 0 or 1. 

But I can only get an accuracy of around 75%.  Am I missing anything?

**Comment by mohaseeb** [comment]
Hello!
You are right, something was broken in the UCR dataset loader; I fixed it (and pushed a new version).
I just ran the example without standardizing the dataset and got 95% accuracy. Can you try to run the example without standardizing the data?
....
epoc=199/199 (iteration=10000) loss=0.189644625859 validation accuracy=0.98
             precision    recall  f1-score   support

        0.0       0.95      0.96      0.95        76
        1.0       0.96      0.95      0.95        74

avg / total       0.95      0.95      0.95       150

[[73  3]
 [ 4 70]]

**Comment by zwang4** [comment]
Hiya, I managed to get 0.93 using the new version. Thanks a lot.

Is there any plan to port this nice tool to Torch/Caffe/Tensorflow to make use of GPUs?

**Comment by mohaseeb** [comment]
Hi @zwang4, yes I am planning to create a Tensorflow implementation, but I don't have an exact data yet.

**Comment by faaizuddin** [comment]
@zwang4 , how did you manage to set the code to predict the labels 0 and 1? In my dataset, there are only 2 labels (1, 2). but somehow it is predicting 0 as well.

**Comment by faaizuddin** [comment]
Also when I use the gunpoint dataset, it has labels 1 and 2, but still this implementation ends up predicting 0s. Doesnt make any sense. Seems to work when I change the labbels to 0 and 1.
